recipes/libftdi/1.x/conanfile.py

```diff
@@ -10,94 +13,86 @@ class LibFtdiConan(ConanFile):
     topics = ("conan", "libftdi1")
```

Comment:
Also, the licenses are not quite correct according to the project's listed home page:

```
    license = "LGPL-2.1-only"
```

Add the additional license, "GPL-2.0-only", in the configure method  only if the EEPROM tool is enabled.

Reply:
According to some of files in the repo, C++ wrapper is GPL as well, but with a special exception as given in the `LICENSE` file.